packages/node_modules/pouchdb-core/src/adapter.js

```diff
@@ -245,6 +245,9 @@ AbstractPouchDB.prototype.putAttachment =
     blob = rev;
     rev = null;
   }
+  if (!type) {
+    console.warn('Attachment', attachmentId, 'on document', docId, 'is missing content_type');
```

Comment:
So to handle the case where `console` is not defined, we have a utility for calling console functions that we should use here https://github.com/pouchdb/pouchdb/blob/master/packages/node_modules/pouchdb-core/src/parseAdapter.js